libraries/stdlib/src/kotlin/reflect/typeOf.kt

```diff
@@ -8,8 +8,7 @@ package kotlin.reflect
 /**
  * Returns a runtime representation of the given reified type [T] as an instance of [KType].
  */
-@Suppress("unused") // KT-12448
 @SinceKotlin("1.3")
 @ExperimentalStdlibApi
-public inline fun <reified T> typeOf(): KType =
+public inline fun <@Suppress("unused") reified T> typeOf(): KType =
```

Comment:
Thanks, but actually this suppress is not needed at all, because the `T` type parameter is references in the docs above.

Reply:
So it would be enough just to remove the suppress from the function.